tools/deploy-to-master-stable-branch.sh

```diff
@@ -32,7 +41,8 @@ echo ""
 
 echo "Building Jetpack"
 npm run distclean
-npm run build
+npm install
+npm run build-production
```

Comment:
Let's use `NODE_ENV=production npm run build` here until we figure out a problem with PO and MO files that this script generates?